api/kroxylicious-filter-api-bom/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.kroxylicious</groupId>
+    <artifactId>kroxylicious-filter-api-bom</artifactId>
+    <version>0.3.0-SNAPSHOT</version>
+
+    <name>Kroxylicious Filter API BOM</name>
+
+    <packaging>pom</packaging>
+    <properties>
+        <jackson.version>2.15.2</jackson.version>
```

Comment:
We need to keep these in sync with the version in the parent, right?   Wonder if dependabot will be sufficiently smart to update both?

Reply:
Now absorbed more of the PR.. so the parent relies on the BOM, so there's no duplication